composer.json

```diff
@@ -21,15 +21,20 @@
         "php": ">=7.1",
         "doctrine/orm": "^2.5",
         "friendsofsymfony/jsrouting-bundle": "^1.6|^2.0",
-        "symfony/framework-bundle": "^3.4|^4.1",
-        "symfony/options-resolver": "^3.4|^4.1",
-        "symfony/property-access": "^3.4|^4.1",
-        "symfony/security": "^3.4|^4.1",
-        "symfony/translation": "^3.4|^4.1",
-        "twig/twig": "^2.9"
+        "symfony/config": "^3.4|^4.1|^5.0",
+        "symfony/dependency-injection": "^3.4|^4.1|^5.0",
+        "symfony/http-foundation": "^3.4|^4.1|^5.0",
+        "symfony/http-kernel": "^3.4|^4.1|^5.0",
+        "symfony/framework-bundle": "^3.4|^4.1|^5.0",
```

Comment:
Do you know if there is any reason to not be compatible with Symfony `4.0`?

Reply:
Change made here for `security reasons`: https://github.com/stwe/DatatablesBundle/pull/885

Not sure if its the responsibility of this bundle to force people to upgrade.
IMH we should keep compatibility with 4.0 and let people upgrade when they can.

Reply:
Dunno, but it's a bit out of scope for this PR ... i've only just expanded the existing composer requirements to 5.0 and added the extra components based on the existing 4.1 requirements for the existing components.